### DISCUSSION TITLE
Unable to focus Markdown widget

### DISCUSSION THREAD

**Discussion by ethael** [Textualize/textual#3635] [Q&A] [answered]
I am encountering a problem, that even when Markdown widget is focusable, I am unable to focus it by just calling focus() method manually. I have prepared a working example with two widgets. A ListView and a Markdown, with a keyboard shortcuts to focus one or another.

https://github.com/ethael/textual-sandbox/blob/main/poc.py

Is this a bug? Or am I missing something?

**Comment by davep** [comment, marked as answer]
I think this might be down to the documentation for `Markdown` being misleading. It says that it is a focusable widget by default, but it isn't. You could make it so by setting its `can_focus` property to `True`, but really you probably want it in something like a `VerticalScroll` so it'll be scrollable if there is more content than can be displayed (and this in turn will allow focus).

**Reply by ethael** [reply]
Thanks a lot for a quick response. Setting `can_focus` to `True` does the trick. I shouldn't relied on the docs and just try it myself. Sorry.
Wrapping in `VerticalScroll` isn't necessary, if you add enough text to my example app, scroll shows automatically and works

**Reply by davep** [reply]
You should be able to rely on the docs! I've already submitted a fix for that, so thanks for highlighting it.
